### PR TITLE
Create LICENSE

### LINKED ISSUE #129: Add LICENCE
Hey, @anmode   I'm active GSSoC'23 contributor and I would like to add a license file to this repository. Please assign me to this issue. Thank you!

### PR DESCRIPTION
**Pull request by Jiggy9**
closes #129

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 anmode
+
```
Comment (anmode) [inline LICENSE:4]: This is wrong! please check the other licenses...and one thing to keep in mind this is a startup project. Thanks

**Review by anmode — COMMENTED** [review]
Need changes

**Comment by anmode** [comment]
Run `npx prettier --write`

**Comment by Jiggy9** [comment]
Hey thank you, but I don't understand why build check fails by vercel? I added file into Github using MIT License template, I'm sorry for causing error.. I would be glad if you guide me. I created other License as well in other projects successfully.

Here's my forked repo which shows MIT License
issue

**Comment by anmode** [comment]
Oh there is a bug..I'll fix it. but the license is not. ok

**Comment by anmode** [comment]
@Jiggy9 ?

**Pull request closed without merging**